e2e_samples/dataset_versioning/insert_sql/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,36 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.163.1/containers/python-3
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": { 
+			"VARIANT": "3.8",
+			// Options
+			"INSTALL_NODE": "true",
+			"NODE_VERSION": "lts/*"
+		}
+	},
+	"settings": { 
+		"terminal.integrated.shell.linux": "/bin/bash",
+		"python.pythonPath": "/usr/local/bin/python",
+		"python.linting.enabled": true,
+		"python.linting.pylintEnabled": true,
+		"python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
```

Comment:
[optional] since you included many linters here, do you also want to include how to use these linters in readme?

Reply:
Umm.. I dont think it's required because
1.  there is no action required from user point of view +
2.  this was set automatically when I installed devcontainer.

Reply:
oh ok, it's an optional comment anyways :D 
I just thought you added how to run unittest(there is no action required from user point of view too), might as well and linting :D